vignettes/extract_convert.Rmd

```diff
@@ -0,0 +1,256 @@
+---
+title: "Parameter extraction and conversion in {epiparameter}"
+output: bookdown::html_vignette2
+bibliography: references.bib
+vignette: >
+  %\VignetteIndexEntry{Parameter extraction and conversion in {epiparameter}}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The reporting of parameters of distributions in the epidemiological literature 
+is varied. In some instances the parameters themselves will be given, in others,
+summary statistics of the distribution are provided, such as mean, standard 
+deviation, variance, among others. 
+
+When working with epidemiological parameters it can be useful to have the values
+of the parameters for common parameterisations of the distribution. For example,
+shape and scale parameters for the gamma and weibull distribution, and the meanlog
+and sdlog parameters for the lognormal distribution. 
+
+In {epiparameter} we provide a range of conversion and extraction functions in
+order to obtain distribution parameters from summary statistics and vice versa.
+
+Here we differentiate between conversion and extraction as what can be analytically
+converted, as conversions, and those that require optimisation as extraction. 
+This vignette aims to describe the functionality of these conversions and 
+extractions provided by the {epiparameter} package.
+
+::: {.alert .alert-info}
+### Conversion versus extraction
+
+Use conversion when possible over extraction to avoid possible limitations associated with
+numerical optimisation used in the extraction function `extract_param()`.
+:::
+
+```{r setup}
+library(epiparameter)
+```
+
+## Conversions
+
+There are two styles of conversion functions in {epiparameter}. The general conversions
+which have the function name `convert_dist_*()` (where `*` and `dist` are placeholders). These convert
```

Comment:
Wouldn't it be simpler for users to have just the one-to-many conversion and they can then easily extract the relevant parameters to reproduce what the one-to-one are doing?

Reply:
I'd given this some thought and decided that the one-to-one conversions, although functionally superfluous, might be easier to use given the function name. So I kept them as the cost of maintenance and increasing the package namespace/API would be outweighed by the simplicity and ease of use of having these functions. 

However, saying that, I do not have a strong preference so if you feel the package would be simpler without them, I'm happy to remove and updated to the vignette to explain how to use the one-to-many function in those cases.

Reply:
Ok, let's park this until #139 is implemented. It should give us a clearer view of how the different exported functions are organized and fit together and help us see what the best way forward is. Does this sound reasonable?

Reply:
Agreed.